packages/awslint/lib/rules/construct.ts

```diff
@@ -184,6 +184,8 @@ constructLinter.add({
   message: 'construct interface must extend core.IConstruct',
   eval: e => {
     if (!e.ctx.interfaceType) { return; }
+    // ICfn interfaces are not construct interfaces
```

Comment:
I wonder if they should be though? `IBucket extends IConstruct`, I wonder if we should `ICfnBucket extends IConstruct` as well.

Reply:
@kaizencc